wiki/608450-SynapseEnvironment.md

```diff
@@ -0,0 +1,96 @@
+## Overview
+
+Synapse is a collaborative platform for researchers and developers. Synapse is
+developed by Sage Bionetworks to offer services that support:
+
+- Organization of digital research assets (e.g., files, SQL tables, Docker
+  images)
+- Recording provenance to give researchers credits for their work
+- Collaboration among researchers
+
+The NLP Sandbox walk in the footsteps of previous successful scientific
+challenges that been organized using Synapse. The NLP Sandbox relies on the
+following features of Synapse to enable its users to develop and benchmark new
+NLP Tools.
+
+- Creation and management of Teams
+- Unlimited storage of private Docker images
+- Benchmark the performance of NLP Tools on public and private datasets
+
+The sections below describe how to setup your development environment on Synapse
+so you can start developing and benchmarking your next NLP tool.
+
+## Create a Synapse account
+
+If you don't have already an account, visit the home page of [synapse.org] and
+register. Make sure to complete your profile information.
+
+## Get Certified
+
+You need to certify your Synapse account if you want to perform the following
+actions:
+
+- Update files to Synapse
+- Update Docker images to Synapse
+- Benchmark the performance of NLP tools
+
+Follow these steps listed below to certify your Synapse account:
+
+1. Go to the page [User Types including Certified Users].
+2. Click on the button "Become a Certified User".
+3. Complete the certification form.
+
+You can confirm that your account is certified by visiting your [Synapse
+Profile].
+
+## Create a Synapse Team
+
+Playing -- huh -- working together will make your experience developing and
+benchmarking NLP tools more enjoyable. Also, at the time of submitting a tool
+for evaluation, Synapse will ask you if the submission is made on behalf of a
+User or a Team. Submitting as a Team offers more flexibility if you decide to
+invite team members to work with you in the future.
+
+Head to your [Synapse Teams page] to
+create a new Team. If you are looking for Team members, checkout the page [NLP
+Sandbox Users & Teams]. You can also re-use an existing team.
+
+## Create a Synapse Project
+
+Synapse projects offer unlimited storage for public and private images, which is
```

Comment:
```suggestion
Synapse projects offer unlimited storage for public and private Docker images, which is
```

Reply:
Its technically not unlimited, because there _is_ a limit on the size of the docker repository that is uploaded into Synapse.

Reply:
I meant unlimited in terms of number of images.